lecture_1/task/go.mod

```diff
@@ -0,0 +1,8 @@
+module code-cadets-2021/lecture_1/07_testing
```

Comment:
You should name your module according to your directory structure.
`module code-cadets-2021/lecture_1/task` instead of `module code-cadets-2021/lecture_1/07_testing`

Reply:
Da, malo samo nepažljivo to copy/pasteao jer mi nije iz nekog razloga radio onaj go mod init